evm/deploy-its.js

```diff
@@ -0,0 +1,250 @@
+require('dotenv').config();
+
+const { getCreate3Address } = require('@axelar-network/axelar-gmp-sdk-solidity');
+const { deployContract, deployCreate3, isAddressArray, writeJSON } = require('./utils');
+const {
+    Wallet,
+    Contract,
+    getDefaultProvider,
+    utils: { defaultAbiCoder, isAddress, keccak256 },
+} = require('ethers');
```

Comment:
why not use hardhat here? seems to be deviating from other scripts

Reply:
seems like a lot of the other deployment scripts are using ethers as well, can change this for all scripts in a separate PR